### PR TITLE
Bump Rails and all SDK versions

### PR DESCRIPTION
**Pull request by SokratisVidros**

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,62 +1,71 @@
...
-gem 'webpacker', '~> 5.0'
-# Turbolinks makes navigating your web application faster. Read more: https://github.com/turbolinks/turbolinks
-gem 'turbolinks', '~> 5'
-# Build JSON APIs with ease. Read more: https://github.com/rails/jbuilder
-gem 'jbuilder', '~> 2.7'
+# Bundle edge Rails instead: gem "rails", github: "rails/rails", branch: "main"
+gem "rails", "~> 7.1.3"
 
+# The original asset pipeline for Rails [https://github.com/rails/sprockets-rails]
 gem "sprockets-rails"
```
Comment (davemo) [inline Gemfile:9]: I think you will also want the `sprockets` gem 🤔 

Screenshot 2024-02-06 at 11 39 11 AM

https://guides.rubyonrails.org/asset_pipeline.html#what-is-the-asset-pipeline-questionmark

File: Gemfile [hunk 2]
```diff
@@ -1,62 +1,71 @@
...
 
-# Windows does not include zoneinfo files, so bundle the tzinfo-data gem
-gem 'tzinfo-data', platforms: [:mingw, :mswin, :x64_mingw, :jruby]
-
-# Loads env variables from env files
-gem "dotenv-rails", groups: [:development, :test]
-
 # Clerk Ruby SDK
-gem 'clerk-sdk-ruby', '~> 2.10.0', require: "clerk"
+gem 'clerk-sdk-ruby', '~> 3.0.0', require: "clerk"
```
Comment (JacobMGEvans) [inline Gemfile:71]:
```suggestion
gem 'clerk-sdk-ruby', '~> 3.0.0', require: "clerk"

```

Comment (JacobMGEvans) [reply Gemfile:71]: That whitespace at the bottom gets rid of the EOF error in GitHub UI

File: app/views/home/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,4 @@
+<main class="main">
+  <%= image_tag("clerk.png", size: 200) %>
+  <a href="https://clerk.com/docs" target="_blank">Clerk documentation</a>
+</main>
```
Comment (JacobMGEvans) [inline app/views/home/index.html.erb:4]:
```suggestion
</main>

```

**Review by JacobMGEvans — APPROVED** [review]
Looks great. As mentioned in internal chat, we should have a formatter config or editor config setup to normalize formatting and styling across the repo and contributors.

**Pull request merged**